src/v/kafka/security/scram_credential.cc

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#include "kafka/security/scram_credential.h"
+
+#include <ostream>
+
+namespace kafka {
+
+std::ostream& operator<<(std::ostream& os, const scram_credential& cred) {
+    fmt::print(
+      os,
+      "salt {} server_key {} stored_key {} iterations {}",
+      cred._salt,
+      cred._server_key,
```

Comment:
Maybe we should obfuscate the key. It's not used anywhere in the patch, but if somebody will use it to log the credentials it's quite possible that the log will be sent somewhere for storage.

Reply:
Good point. I've removed all of this sort of stuff from logging.